Cargo.toml

```diff
@@ -199,6 +199,7 @@ indicatif = "0.17.7" # Progress bar
 infer = "0.16" # infer MIME type by checking the magic number signaturefer MIME type by checking the magic number signature
 insta = "1.23"
 itertools = "0.13"
+jiff = "0.1.15"
```

Comment:
We already use `time` to format timestamps - should we switch to `jiff` everywhere?

Reply:
I just wanted to try it. It's definitely much nicer to use, but maybe not worth it?

Reply:
Replaced with `time`